run-test.sh

```diff
@@ -157,6 +157,8 @@ runtest()
   fileName="${file##*/}"
   fileNameWithoutExtension="${fileName%.*}"
   testDllName="$fileNameWithoutExtension.dll"
+  local LowerOS="$(echo $OS | awk '{print tolower($OS)}')"
```

Comment:
Since we've used this twice, would you move LowerOS out of the two functions that use it and up to where the rest of the main variables are set?